# A Practical Introduction to Genomics, Transcriptomics, and more - KAUST 2019

## Table of Content
  * [About](https://roblehmann.github.io/hts_workshop2019/#about)
  * [Program](https://roblehmann.github.io/hts_workshop2019/#program)
  * [Speaker](https://roblehmann.github.io/hts_workshop2019/#speaker)
  * [Location](https://roblehmann.github.io/hts_workshop2019/#Location)
  * [Contacts](https://roblehmann.github.io/hts_workshop2019/#Contacts)
## About

Genomics and transcriptomics are rapidly becoming standard methods to examine living systems. This workshop provides an introduction to the methods and tools used to analyse next-generation-sequencing data, provides understanding of its limitations and a basic understanding on how the analysis algorithms work. This workshop will take the participants through an example project demonstrating how to evaluate the quality of data as provided by a sequencing facility, how to perform a de-novo genome assembly, and how to align the data against a known and annotated reference genome. Furthermore, participants will learn how to compare transcription data between different samples.

The workshop is scheduled for 18th - 22nd of August, from 9:30 to 17:00, and will consist of seminars intertwined with hands-on sessions and discussions. The course is open for all students, postdocs and research scientists of BESE division.
 
Major topics to be covered: Genome assembly, Transcriptomics, Visualization and Tips/tricks.

Please register **[here](https://www.surveymonkey.com/r/SRBQK6K)** and fill out the short pre-survey. The link contains a short summary of the course, pre-requisites and 7 questions that needs your response.



**THE LAST DATE FOR REGISTRATION IS 1st AUGUST 2019.**

Acceptance mails, course syllabus, venue and other details will be sent after the deadline.

## Program
### Sunday - 18th of August: The Tools of Computation Biology
1. Introduction to programming basics
2. Utility commands in data management
3. Common file formats in NGS data
4. Introduction to High-Performance Computing environment

### Monday - 19th of August: Where do the data come from?
1. Introduction to Next-Generation-Sequencing
2. Quality control on raw data 
3. Assembling a small genome sequence

### Tuesday - 20th of August: How to make a genome
3. Assembly QC
4. Annotating genes 
5. Using long reads for assembly
6. Hybrid Assembly QC

### Wednesday - 21th of August: How to quantify transcription
1. Experimental design and strategies
2. Data cleaning + mapping
3. Quantification
4. Visualization 

### Thursday - 22th of August: How to compare transcription between samples
5. Differential expression
6. Functional annotation
7. Alternative splicing
8. Summary + QA + Recap

## Speaker
  * [Arun Nagarajan](https://rsrc.kaust.edu.sa/Pages/Nagarajan.aspx), PostDoc at KAUST, is interested in understanding the molecular mechanism of acclimatization to stress by a biological organism using new-age technology. 
  * [Robert Lehmann](https://livingsystems.kaust.edu.sa/Pages/Robert%20Lehmann.aspx), Research Scientist at KAUST, integrates genetic and transcriptomic data to gain insights into genetic responses to stress in established model organisms as well as new marine model organisms.
  * [Octavio Salazar Moya](https://rsrc.kaust.edu.sa/Pages/Salazar%20Moya.aspx), PostDoc at KAUST, is interested in increasing food production, particularly by increasing salt tolerance of plants.
  * [Manjula Thimma](https://livingsystems.kaust.edu.sa/Pages/Thimma.aspx), Research Scientist at KAUST, is interested in understanding the role of retrotransposons in cell identity, reprogramming and differentiation.
  * [Alaguraj Veluchamy](https://chromatin.kaust.edu.sa/Pages/AlagurajVeluchamy.aspx), Research Scientist at KAUST, is interested in **TBA**

## Location
**TBA**

## Contacts

  * [Arun Nagarajan](arun.nagarajan@kaust.edu.sa)
  * [Manjula Thimma](manjula.thimma@kaust.edu.sa)
  * [Robert Lehmann](robert.lehman@kaust.edu.sa)
